Tutorials/TurnBasedRPG/source/CombatHUD.hx

```diff
@@ -157,6 +158,7 @@ class CombatHUD extends FlxTypedGroup<FlxSprite>
 	 */
 	public function initCombat(PlayerHealth:Int, E:Enemy):Void
 	{
+		_sprScreen.drawFrame();
```

Comment:
I guess that's what I was missing.. Why does setting dirty to true not work anymore?